app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1389,26 +1355,114 @@
               "units-kilowatt-hours"
             ]
           }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "fuel-sales-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "fuel-sales-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-passenger-vehicles"
+          },
+          {
+            "id": "fuel-sales-on-road-transport-activity",
+            "on-road-transport-vehicle-type": "vehicle-type-public-transport-vehicles"
+          }
         ]
       },
       {
         "id": "induced-activity-transboundary-trips-on-road-transport-methodology",
+        "inputRequired": [
+          "total-distance-travelled-in-in-boundary-trips",
+          "total-distance-travelled-in-transboundary",
+          "fuel-intensity"
+        ],
         "disabled": true
       },
       {
         "id": "induced-activity-only-departing-trips-on-road-transport-methodology",
-        "disabled": true
+        "disabled": true,
+        "inputRequired": [
+          "total-distance-travelled-in-in-boundary-trips",
+          "distnace-travelled-within-city-transboundary-departing-only",
+          "fuel-intensity"
+        ]
       },
       {
         "id": "geographic-on-road-transport-methodology",
-        "disabled": true
+        "inputRequired": [
+          "total-distance-travelled-in-in-boundary-trips",
+          "fuel-intensity"
+        ],
+        "activities": [
+          {
+            "id": "geographic-on-road-transport-activity",
+            "unique-by": [
+              "on-road-transport-vehicle-type",
+              "on-road-transport-fuel-source"
+            ],
+            "activity-title": "total-distance-travelled-in-boundary",
+            "units": ["units-kilometers", "units-miles"],
+            "extra-fields": [
+              {
+                "id": "on-road-transport-vehicle-type",
+                "options": [
+                  "vehicle-type-all",
+                  "vehicle-type-passenger-vehicles",
+                  "vehicle-type-commercial-vehicles",
+                  "vehicle-type-public-transport-vehicles",
+                  "vehicle-type-emergency-vehicles",
+                  "vehicle-type-service-vehicles"
+                ]
+              },
+              {
+                "id": "on-road-transport-fuel-type",
+                "options": [
+                  "fuel-type-gasoline",
+                  "fuel-type-diesel",
+                  "fuel-type-lpg",
+                  "fuel-type-cng",
+                  "fuel-type-biofuel"
+                ]
+              },
+              {
+                "id": "on-road-transport-fuel-intensity",
+                "type": "number",
+                "units": [
+                  "units-grams-per-kilometer",
```

Comment:
units for fuel intensity:
- tonnes-per-kilometer
- gallons-per-kilometer
- cubic-meters-per-kilometer
- tonnes-per-miles
- gallons-per-miles
- cubic-meters-per-miles